01-shapefile-attributes.Rmd

```diff
@@ -101,13 +102,12 @@ point <- readOGR("boundaryFiles/HARV/", "HARVtower_UTM18N")
 
 ```
 
+##Querying shapefile metadata 
 Recall from the previous lesson that shapefile metadata can include a _class_, 
```

Comment:
This was the coding that was already in the file.  They seem to function the same as ``.  I'm happy to use `` only